website/blog/2024-04-15-0.74-post.md

```diff
@@ -28,9 +30,94 @@ Today we're releasing React Native 0.74!...
   - Contextualize what changes are coming into 0.74 from proposal
 - Warning on using react-native init https://github.com/facebook/react-native/commit/a6397afb7f4279e70afa025fe6f9fa0b8e3de3fd
 
-## Yoga 3.0
+## New layout behaviors
 
-- Link to Yoga post, need clarification on how it affects users.
+React Native 0.74 includes [changes to Yoga](https://yogalayout.dev/blog/announcing-yoga-3.0) which enable more correct layout. These changes make styling more predictable, and enable rendering components written for the web with greater fidelity.
+
+React Native intentionally preserves some incorrect layout behaviors, where fixing them was found to effect a signigicant number of real-world components. Layout conformance will be able to be configured more granularly in future versions of React Native.
```

Comment:
I debated including this part:

> Bleeding edge users on the New Architecture can test the effects of rendering their components in a mode which prioritizes correctness and Web compatibility over compatibility with existing React Native components. This API will change in a future version of React Native.
>
> ```tsx
> import {} from 'react-native/types/experimental'
> 
> // This component tree inherits layout which acts like web
> <View experimental_layoutConformance="strict">
>   <Widget />
> </View>
> 
> // This component tree inherits layout which acts like React Native
> <View experimental_layoutConformance="classic">
>   <Doodad />
> </View>
> ```

Reply:
I think that might be interesting to add in a "Tip" clause? Or perhaps an "Info" admonition https://docusaurus.io/docs/next/markdown-features/admonitions

Reply:
I think I will avoid it for now until it can be contextualized better.